AirRobeDemo/AirRobeDemo/Controllers/ConfirmationViewController.swift

```diff
@@ -19,7 +19,8 @@ final class ConfirmationViewController: UIViewController {
         super.viewDidLoad()
         airRobeConfirmation.initialize(
             orderId: orderId,
-            email: email
+            email: email,
+            orderSubTotalCents: 10000
```

Comment:
"SubTotal" is one word, so should be "orderSubtotalCents" like in the docs below
https://connector.airrobe.com/docs/ios